### ISSUE TITLE
Monotonic option for ncrcat

### ISSUE THREAD

**Issue by rsignell-usgs** [nco/nco#17]
We often encounter files that we want to concatenate along the record dimension, but that have overlapping time values.   It would be great to have a `--monotonic` option for `ncrcat`.  

 I realize that this means you would have to read the coordinate variable in the record dimension, but since NCO allows slicing now on coordinate variables that should be straightforward, no?  :smiley_cat: 

The default behavior should be that later files overwrite the previous files when overlapping coordinate records are found.

**Comment by czender** [comment]
this is ambitious. it would be easier to alter the code so that it never wrote non-monotonic records. preferring values from later files means erasing data already written, or some other fairly significant change, like pre-reading the record coordinate before writing anything...

**Comment by rsignell-usgs** [comment]
Yeah, I was thinking of working from the last file to the first, creating a monotonic time coordinate and figuring out start/stop time indices for each file before writing anything.
